docs/admin/install/vm-install/ubuntu.md

```diff
@@ -131,6 +131,22 @@ sudo /opt/jans/jans-cli/jans_cli_tui.py
 
 Full TUI documentation can be found [here](../../config-guide/config-tools/jans-tui/README.md)
 
+If you have selected casa during installation you can access casa using url ```https://<host>/jans-casa```
+
+## Let's Encrypt
+ To generate Let’s Encrypt CA certificate run below commands and add required information at prompt:
+```bash 
+sudo apt update && sudo apt install certbot python3-certbot-apache
+sudo certbot --apache -d FQDN
+```
+to check certbot status
+```
+sudo systemctl status certbot.timer
+```
+to renew certificate run
+```
+sudo certbot renew --dry-run
+```
```

Comment:
@manojs1978 For Ubuntu, don't we have to update values in `https_jans.conf` ?

Reply:
no it will automatically done